jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -429,7 +433,8 @@ def state_attr_for_line_visible(state):
 
     @property
     def user_api(self):
-        expose = ['multiselect', 'viewer', 'layer', 'select_all', 'subset_visible']
+        expose = ['multiselect', 'viewer', 'layer', 'select_all', 'subset_visible',
+                  'set_histogram_nbins', 'set_histogram_x_limits', 'set_histogram_y_limits']

```

Comment:
Maybe we should leave the x/y-limits out of the API for now since they may change when we implement interactive zooming...?